src/popup/router/components/AddAccountCard.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div class="add-account-card">
+  <div
```

Comment:
In my opinion using plain `div` with the ability to click on it is an antipattern: https://stackoverflow.com/a/32659785/2399521. I think it should be a link or button or at least have the tabindex.